### PR TITLE
MEDIA-1941: scaffold gathering barbelling stats

### PR DESCRIPTION
**Pull request by reddvl1980**

### CODE REVIEW

**Review by RicardoMDomingues — APPROVED** [review]
LGTM

File: bridge/engine/Engine.h [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "bridge/engine/EngineStats.h"
+#include "bridge/Stats.h"
```
Comment (olofkallander) [inline bridge/engine/Engine.h:3]: do we need this change?

File: bridge/engine/EngineMixer.h [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #include "api/SimulcastGroup.h"
 #include "bridge/engine/ActiveTalker.h"
 #include "bridge/engine/BarbellEndpointMap.h"
-#include "bridge/engine/EngineStats.h"
+#include "bridge/Stats.h"
```
Comment (olofkallander) [inline bridge/engine/EngineMixer.h:6]: revert?

**Review by olofkallander — CHANGES_REQUESTED** [review]

File: bridge/engine/Engine.h (hunk 1 above)
Comment (reddvl1980) [reply bridge/engine/Engine.h:3]: We used to, when stats gathering was in the Engine. Now not. I'll revert.

File: bridge/Stats.cpp [hunk 3]
```diff
@@ -466,6 +466,52 @@ ConnectionsStats SystemStatsCollector::collectLinuxNetStat(uint16_t httpPort, ui
...
+    result["octets"] = thiz.octets;
+    result["packets"] = thiz.packets;
+    result["lostPackets"] = thiz.lostPackets;
+    result["bitrateKbps"] = thiz.bitrateKbps;
+    result["packetsPerSecond"] = thiz.packetsPerSecond;
+    result["activeStreamCount"] = thiz.activeStreamCount;
+    return result;
+}
+
+nlohmann::json BarbellStatsToJson(const BarbellStats& thiz)
```
Comment (olofkallander) [inline bridge/Stats.cpp:483]: barbellStatsToJson

File: bridge/Stats.cpp [hunk 4]
```diff
@@ -466,6 +466,52 @@ ConnectionsStats SystemStatsCollector::collectLinuxNetStat(uint16_t httpPort, ui
...
+    audio["outbound"] = barbellPayloadStatsToJson(thiz._stats[BarbellStats::AUDIO_SEND]);
+    audio["inbound"] = barbellPayloadStatsToJson(thiz._stats[BarbellStats::AUDIO_RECV]);
+    video["outbound"] = barbellPayloadStatsToJson(thiz._stats[BarbellStats::VIDEO_SEND]);
+    video["inbound"] = barbellPayloadStatsToJson(thiz._stats[BarbellStats::VIDEO_RECV]);
+    result["audio"] = audio;
+    result["video"] = video;
+    return result;
+}
+
+nlohmann::json MixerBarbellStatsToJson(const MixerBarbellStats& thiz)
```
Comment (olofkallander) [inline bridge/Stats.cpp:497]: mixerBarbellStatsToJson

**Review by olofkallander — APPROVED** [review]
2 changes requested

**Pull request merged**